### ISSUE TITLE
4.0.0 tests fail because of missing tests/conftest.py

### ISSUE THREAD

**Issue by yurivict** [dimazest/google-ngram-downloader#13]
tests expect fixtures bigrams_indices,bigrams_indices,records which aren't included in 4.0.0.

Please make a release with tests working.

**Comment by dimazest** [comment]
Could you run tests as ``python setup.py test`` and post the error message, please.

**Comment by yurivict** [comment]
Using version 4.0.0:
```
test/test_main.py .....
test/test_util.py EEE..E

================================================================================== ERRORS ==================================================================================
________________________________________________________________ ERROR at setup of test_get_indices_1grams _________________________________________________________________
file /usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py, line 10
  def test_get_indices_1grams(unigram_indices):
E       fixture 'unigram_indices' not found
>       available fixtures: cache, capfd, capsys, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py:10
_______________________________________________________________ ERROR at setup of test_get_indices_manygrams _______________________________________________________________
file /usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py, line 17
  def test_get_indices_manygrams(bigrams_indices):
E       fixture 'bigrams_indices' not found
>       available fixtures: cache, capfd, capsys, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py:17
________________________________________________________________ ERROR at setup of test_get_indices_5grams _________________________________________________________________
file /usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py, line 24
  def test_get_indices_5grams(bigrams_indices):
E       fixture 'bigrams_indices' not found
>       available fixtures: cache, capfd, capsys, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py:24
_________________________________________________________________ ERROR at setup of test_count_coccurrence _________________________________________________________________
file /usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py, line 62
  def test_count_coccurrence(records):
E       fixture 'records' not found
>       available fixtures: cache, capfd, capsys, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/usr/ports/misc/py-google-ngram-downloader/work/google-ngram-downloader-4.0.0/test/test_util.py:62
==================================================================== 7 passed, 4 error in 0.65 seconds =====================================================================
*** Error code 1
```

Failing fixtures aren't defined in 4.0.0 source tree, but the y appear in the github version.

**Comment by dimazest** [comment]
Did you get the 4.0.0 trough pypi or github release? I suppose you didn't clone the repo.

**Comment by dimazest** [comment]
please check 4.0.1 from pypi

**Comment by yurivict** [comment]
Thanks, it is fixed in 4.0.1.
